src/app/trip/[id]/trip.css

```
@@ -0,0 +1,336 @@
+    display: grid;
+    gap: 8px;
+    max-height: min(50%, 800px);
+    width: min(1200px, calc(100%));
+}
+.map-overlay .map-dialog {
+    background: white;
+    border-radius: 8px;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, 0.1);
+    padding: 16px;
+    /* width: 1200px or 100%, dotiraka chiisaii hou */
+    width: max(calc(100%-32px), 1200px); 
```

Biome (2.1.2)

[error] Unexpected shorthand property inset after top



(lint/suspicious/noShorthandPropertyOverrides)